### PR TITLE
resolve undefined latestPathname bug

### PR DESCRIPTION
**Pull request by dev-tronghieu**

### CODE REVIEW

File: apps/web/src/state/app/settings.ts [hunk 1]
```diff
@@ -21,7 +21,8 @@ export const setPrivacy = async (hideBalance: boolean) => {
 
 export const setPathname = async (latestScreen: string) => {
 	await modules.storage.upsert<SettingDocument>(id, async (doc) => {
-		doc.config.latestLocation = latestScreen;
+		if (doc.config.latestLocation !== latestScreen)
+			doc.config.latestLocation = latestScreen;
```
Comment (fluctlight-kayaba) [inline apps/web/src/state/app/settings.ts:25]: seems not related to `undefined` stuff

Comment (dev-tronghieu) [reply apps/web/src/state/app/settings.ts:25]: With the accounts using the old database, there is no field name `latestLocation` in `doc.config` which causes `undefined latestLocation` error
